package/obs/rmt-server.changes

```diff
@@ -1,3 +1,8 @@
+- Adding Uptime tracking capability
+  * Uptime data is included in the data sent to SCC via RMT
```

Comment:
Please add here the relevant bugzilla(bsc)/jira(ped) tickets here. We can cleanup the rest when preparing for rmt 2.17.